specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2022-06-15/EventGrid.json

```diff
@@ -0,0 +1,10924 @@
+      "post": {
+        "tags": [
+          "PartnerTopics"
+        ],
+        "summary": "Deactivate a partner topic.",
+        "description": "Deactivate specific partner topic.",
+        "operationId": "PartnerTopics_Deactivate",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group within the user's subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "partnerTopicName",
+            "in": "path",
+            "description": "Name of the partner topic.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PartnerTopic"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
+          "PartnerTopics_Deactivate": {
+            "$ref": "./examples/PartnerTopics_Deactivate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventGrid/{parentType}/{parentName}/privateEndpointConnections/{privateEndpointConnectionName}": {
+      "get": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "summary": "Get a specific private endpoint connection.",
+        "description": "Get a specific private endpoint connection under a topic, domain, or partner namespace.",
+        "operationId": "PrivateEndpointConnections_Get",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group within the user's subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parentType",
+            "in": "path",
+            "description": "The type of the parent resource. This can be either \\'topics\\', \\'domains\\', or \\'partnerNamespaces\\'.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "topics",
+              "domains",
+              "partnerNamespaces"
+            ],
+            "x-ms-enum": {
+              "name": "ParentType",
```

Comment:
Please align all the occurrence of this enum type name to "PrivateEndpointConnectionsParentType" with previous api-versions, such like https://github.com/Azure/azure-rest-api-specs/blob/2d6cb29af754f48a08f94cb6113bb1f01a4e0eb9/specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2021-12-01/EventGrid.json#L2642
This change of name is introducing an unneccesary breaking change to all the SDKs. And the old name is more descriptive